sml_small/editing/totals_and_components/totals_and_components.py

```diff
@@ -396,13 +427,13 @@ def totals_and_components(
 
 
 def initialize_components_list(
-    component_list: List[float],
+    component_list: List[Decimal],
 ) -> List[ComponentPair]:
     """
     Takes the list of components values and constructs Component_List objects from them
 
     :param component_list: List of component values, the sum of which, should match the received total value.
-    :type component_list: list(float)
+    :type component_list: list(decimal)
```

Comment:
Should probably be Decimal to remain consistent

Reply:
It is already decimal isn't it?

Reply:
Capital D for decimal, like the other DocStrings you changed

Reply:
Ah i see what you mean. Thanks Steve